dist/commands/launch/client/placeholder.js

```diff
@@ -15,13 +15,30 @@ GNU General Public License for more details.
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
+
+const { createHash } = require("crypto");
+
+function generateUUID(playerName) {
+    const data = "OfflinePlayer:" + playerName;
+    const hash = createHash("md5").update(data, "utf8").digest();
+    hash[6] &= 0x0f; hash[6] |= 0x30; 
+    hash[8] &= 0x3f; hash[8] |= 0x80;
+    return [...hash].map((b, i) =>
+    (b & 0xff).toString(16).padStart(2, "0") +
+    ([3, 5, 7, 9].includes(i) ? "-" : "")
+    ).join("").slice(0, 36);
+}
+

```

Comment:
I think this should be moved to a different file. Don't think placeholder.js should be responsible for generating the UUID.